### PR TITLE
Move demo to cerebral-provider-forms

### PR DESCRIPTION
**Pull request by edgesoft**

### CODE REVIEW

File: demos/demo/src/modules/user/index.js [hunk 1]
```diff
@@ -1,39 +1,45 @@
...
       },
       password: {
         value: '',
         validationRules: ['minLength:5'],
-        validationMessages: ['loginValidationPasswordTooShort'],
+        validationMessages: {
+          'minLength': 'loginValidationPasswordTooShort'
+        },
         isRequired: true,
         requiredMessage: 'loginValidationPasswordRequired'
```
Comment (henri-hulski) [inline demos/demo/src/modules/user/index.js]: requiredMessage doesn't exist anymore. You have to check in the view for `isRequired && !hasValue` or similar. 
You must also prevent now showing the error message on empty field when field is required (maybe a bug?). 
I don't use a required message anymore but just change the color of the field to indicate an error.

File: demos/demo/src/modules/user/index.js [hunk 2]
```diff
@@ -1,39 +1,45 @@
...
+    $signIn: {
       email: {
         value: '',
         validationRules: ['isEmail'],
-        validationMessages: ['loginValidationEmailNotValid'],
+        validationMessages: {
+          'isEmail': 'loginValidationEmailNotValid'
+        },
         isRequired: true,
         requiredMessage: 'loginValidationEmailRequired'
```
Comment (henri-hulski) [inline demos/demo/src/modules/user/index.js]: See below.

File: demos/demo/src/modules/user/index.js (hunk 1 above)
Comment (edgesoft) [inline demos/demo/src/modules/user/index.js]: @henri-hulski But yeah! Should remove requiedMessage 👍

Comment (henri-hulski) [inline demos/demo/src/modules/user/index.js]: Yeah but for an empty field it should not show the error message (eg. "This is not a correct email address"). And it does only if the field is required. This was not the case with cerebral-forms.

Comment (henri-hulski) [inline demos/demo/src/modules/user/index.js]: But ok since requiredMessage is gone maybe that makes sense.
You can always override the behavior.

Comment (edgesoft) [inline demos/demo/src/modules/user/index.js]: Yes, I will remove the requiredMessage. This is the solution that the writers of demo took 
```js
const showError = field => signIn.showErrors && !field.isValid
```
I will let this be 👍

Comment (henri-hulski) [inline demos/demo/src/modules/user/index.js]: I was actually adding cerebral-forms to the demo. ;-)

Comment (henri-hulski) [inline demos/demo/src/modules/user/index.js]: So in cerebral-forms the behavior was that if a required field was empty the requiredMessage was shown. This is not the case anymore. 

BTW I was talking about a bug in cerebral-provider-forms not the demo.

**Comment by edgesoft** [comment]
The french translation didn't have the error messages so the english is there.

**Pull request merged**